eva-cli/src/main.rs

```diff
@@ -125,18 +125,28 @@ fn dispatch(inputs: &ArgMatches, configuration: &Configuration) -> Result<()> {
             Ok(set_field(configuration, field, id, value)?)
         }
         ("tasks", Some(_submatches)) => {
-            let tasks = block_on(eva::tasks(configuration))?;
-            println!("Tasks:");
-            for task in &tasks {
-                // Indent all lines of task.pretty_print() by two spaces
-                println!("  {}", task.pretty_print().split("\n").join("\n  "));
+            if block_on(eva::num_tasks(configuration))? == 0 {
+                println!("No tasks found in the database!")
```

Comment:
Cool, and can we also add something like "You can add a task by calling `eva add <content> <deadline> <duration> <importance>`." or something along those lines? What do you think?

Reply:
Sure thing, I like that! Will update.

Reply:
Sweet!